docs/contributing-via-pr.md

```diff
@@ -0,0 +1,42 @@
+# Submitting your Operator via Pull Requests (PR) in community operators project
+
+## Overview
+To submit an operator one has to do these steps
+
+1. Fork project `https://github.com/operator-framework/community-operators`
+1. Make a pull request
+1. Place the operator in the target directory. [More info](./contributing-where-to.md)
+    - community-operators (Openshift operator)
+    - upstream-community-operators (Kubernetes operator)
+1. Configure `ci.yaml` file. [More info](./operator-ci-yaml.md)
+    - Setup reviewers
+    - Operator versioning strategy
+1. Verify tests and fix problems, if possible
+1. Ask for help in the PR in case of problems
+
+
+## Pull request
+
+When a pull request is created, a number of tests are executed via community hosted pipeline. One can see the results in the comment section of conversation tab.
+
+![PR](images/op_test_pr.png)
+
+## You are done
+User is done when all tests are green. When the PR is merged, the community release pipeline will be triggered.
+
+## Test results failed?
+When operator tests are failing, one can see a following picture
+
+![Summary of test results when failing](images/op_pr_tests_failed.png)
+
+In case of failures, please have a look at the logs of specific tests. If an error is not clear to you, please ask in the PR. Maintainers will be happy to help you with it.
+
+## Useful commands interacting with the pipeline
+You can post the following comment/command:
+
+Command | Functionality |
+--- | --- | 
+`/hold` | Setting PR on hold to prevent merging |
```

Comment:
I don't think the `hold` commands are supported with a new pipeline. Have you tried it?

Reply:
I tested it and looks like this command is not supported in the new community operator pipeline, hence removing it.